learn/experimental/vector_search.mdx

```diff
@@ -0,0 +1,14 @@
+
+# Vector search
+
+Vector search is an experimental technology that uses Large Language Models to retrieve search results. This feature is useful when you are interested in results based on the meaning and context of a query, such as suggesting related products in webshop or articles in an online magazine.
+
+[Visit the GitHub discussion to learn how to use vector search](https://github.com/meilisearch/product/discussions/621).
+
+Are you using this feature? Meilisearch wants to hear from you! Tell us about your experience in the [GitHub discussion](https://github.com/meilisearch/product/discussions/621). All feedback is useful and helps make Meilisearch better and easier to use.
```

Comment:
```suggestion
Are you using this feature? We want to hear from you! Tell us about your experience in the [GitHub discussion](https://github.com/meilisearch/product/discussions/621). All feedback is useful and helps make Meilisearch better and easier to use.
```

Idk why but I always find it creepy when text personifies companies in this way. "Meilisearch wants to hear from you!" 😈